itou/www/companies_views/urls.py

```diff
@@ -7,6 +7,7 @@
 app_name = "companies_views"
 
 urlpatterns = [
+    path("company_home", views.company_home, name="company_home"),
```

Comment:
Les URLs et vues sont déjà dans des espaces de noms donc pas besoin de rajouter (et donc dupliquer) le "company".
Et je réserverais l'utilisation de "home" uniquement aux pages d'accueil, ça évitera des incompréhensions.

Proposition :
```suggestion
    path("overview", views.overview, name="overview"),
```

Reply:
Je n'avais rien trouvé d'autre que ce company_home. Bien mieux "overview"